packages/grid_client/scripts/connection.ts

```diff
@@ -0,0 +1,64 @@
+/* 
+  Use this script to test grid connection:
+  This script creates 2 instances of the grid client, then after 10 minutes, it performs a query on both instances.
+
+  Steps:
+    1 - Connect to the grid
+    2 - Close the internet connection
+    3 - After 9 minutes, open the internet connection again
+  The grid should automatically reconnect and request the twin ID.
+*/
+
+import { GridClient, NetworkEnv } from "../src";
+
+// Replace with actual mnemonics
+const clientMnemonic1 = "your_mnemonic_here";
+const clientMnemonic2 = "your_mnemonic_here";
+
+// Choose the desired network (main, test, qa, dev)
+const NETWORK = NetworkEnv.dev;
+
+// Function to log the current minute along with the client's name
+function logCurrentMinute(clientName: string, time = 1) {
+  console.log(`${clientName} running count ${time}`);
+}
+
+// Client 1 logic
+async function newClient(mnemonic: string, clientNumber: number) {
+  const options = {
+    network: NETWORK,
+    mnemonic: mnemonic,
+  };
+
+  let time = 1;
+
+  try {
+    const gridClient = new GridClient(options);
+    await gridClient.connect();
+
+    // Log the current minute every minute
+    setInterval(() => {
+      time += 1; // script starts at the first minute, then after one minute will log the second..etc
```

Comment:
you already started the variable with 1 and after increasing it here, the first log will be 2 mins while it should be 1.